### DISCUSSION TITLE
2.4 update on rune words?

### DISCUSSION THREAD

**Discussion by Dicska** [fabd/diablo2-runewizard#24] [2. GENERAL CHAT]
Hello,

I'm just curious if there's any work being done (or planned) for updating rune words to the 2.4 patch changes. Absolutely no push or anything, I just would like to know if it's worth checking back here and there.

Thank you for all the work, the tool is amazing.

**Comment by fabd** [comment]
Hey there,

Yes, I would like to update the site to D2R patch 2.4.

Unfortunately I have been unable to experience D2R yet; I hope to dust off my gaming PC in May - I am stuck on a Mac Mini atm for development purposes.

I may be able to update the site in the meantime though.

Looking at the patch notes : https://news.blizzard.com/en-us/diablo2/23788293/diablo-ii-resurrected-patch-2-4-now-live

From what I gather the only changes related to Runewizard:

- 4 runewords have item type requirements updated
- 7 entirely new * ladder only * runewords

... that's much less than I expected.

Because I have not been able to play D2R it's not yet clear to me if you can still run the last "classic" D2 patch? I assume if you play "Classic" in D2R it just means 4 acts, but you're still playing 2.4 right?

Either way I think I may want to add a checkbox ; which could be enabled by default ; to enable D2R changes. Unchecking it zould allow to see Runewords from 1.13c.

That said I don't know if it is a viable long term strategy. Handling old vs new runewords is easy enough;, but with more upcoiming changes I could also just ditch the old 1.13c data... I don't know :/ - It's really hard to tell as I have almost zero feedback from Runewizard visitors.

I guess I'll try to support 1.13c Runewords + 2.4 Runewords first as it should be easy enbough and if it gets more complex in future I'll just drop the old data in favor of D2R.

....

Anyway... 😅  back to your question... I'd say check back mid May and end of May hopefully by then I will have updated the site.

**Comment by fabd** [comment]
@Dicska Site is updated. Please let me know if you find any typos / errors.

I just copy pasted descriptions from  https://news.blizzard.com/en-us/diablo2/23788293/diablo-ii-resurrected-patch-2-4-now-live#rune

These do not specify the level requirement, assumed to be 1.

**Reply by Dicska** [reply]
> Site is updated. Please let me know if you find any typos / errors.

Yaaaay, thank you for all your hard work, you're great!
I found some actual level requirements on d2runewizard.com.

**Reply by fabd** [reply]
Okay I just updated the site with the level requiremetns.

**Reply by Dicska** [reply]
\o/ <3

**Comment by fabd** [comment]
You may double check the data here:

https://github.com/fabd/diablo2-runewizard/blob/main/src/data/runewords.ts#L507


And here

https://github.com/fabd/diablo2-runewizard/blob/main/src/data/runewords-descriptions.ts#L1030
